lib/fs.js

```diff
@@ -268,6 +269,26 @@ function handleError(val, callback) {
   return false;
 }
 
+function openWrap(path, flags, mode, req) {
+  var longPath = pathModule._makeLong(path);
+  if (isWindows) {
+    // \\?\c:\somepath\1:path
+    // \\?\UNC\somepath\2:path
+    if (longPath.indexOf(':', 6) !== -1) {
```

Comment:
If we're going to start explicitly checking for these, perhaps we should be checking for *all* of the reserved characters, either here in JS or at the libuv layer.